examples/overlay/messages/10_output_jaccard.json

```diff
@@ -0,0 +1,122 @@
+                "object": "protein"
+            },
+            "j1": {
+                "subject": "drug",
+                "predicate": "biolink:has_jaccard_index_with",
+                "object": "type-2 diabetes"
+            }
+        }
+    },
+    "knowledge_graph": {
+        "nodes": {
+            "MONDO:0005148": {"name": "type-2 diabetes"},
+            "CHEBI:6801": {
+                "name": "metformin",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "CHEBI:5441": {
+                "name": "glyburide",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "UniProtKB:Q92786": {
+                "name": "PROX1",
+                "category": "biolink:Protein"
+            },
+            "UniProtKB:P30279": {
+                "name": "CCND2",
+                "category": "biolink:Protein"
+            }
+        },
+        "edges": {
+            "k7g89spc": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:P30279"
+            },
+            "df87ff82": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:Q92786"
+            },
+            "5133c100": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:P30279"
+            },
+            "vk456hb1": {
+                "subject": "UniProtKB:Q92786",
+                "predicate": "biolink:correlated_with",
+                "object": "MONDO:0005148"
+            },
+            "lkg89002": {
+                "subject": "UniProtKB:P30279",
+                "predicate": "biolink:correlated_with",
+                "object": "MONDO:0005148"
+            },
+            "j1_1": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:has_jaccard_index_with",
+                "object": "MONDO:0005148",
+                "attributes": [
+                    {
+                      "name": "jaccard_index",
+                      "value": "0.66",
+                      "type": "EDAM:data_1772",
+                      "url": null,
+                      "source": null
+                    }
+                ]
+            },
+            "j1_2": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:has_jaccard_index_with",
+                "object": "MONDO:0005148",
+                "attributes": [
+                    {
+                      "name": "jaccard_index",
+                      "value": "0.33",
+                      "type": "EDAM:data_1772",
+                      "url": null,
+                      "source": null
+                    }
+                ]
+            }
+        }
+    },
+    "results": [
```

Comment:
Why no `j1_2` in the results?

Reply:
So that was to show what would happen if some things were left out of the results before running a jaccard overlay like if a filter had been run and removed it ect. 

I.e. it will still add the edges for everything in the knowledge graph but will only add the new bindings to the results if the corresponding nodes are already included into the results. Does that make sense?

Reply:
Ah, I see. I still think it may be helpful somewhere to capture the actual workflow that was run to transform the input to the output.

Reply:
The workflow transforming input to output is just the jaccard overlay, right? How the input was generated is a separate issue... This is perhaps not the simplest possible input example, but I think everything happening here is clear and sensible.